### ISSUE TITLE
Python unittest not working

### ISSUE THREAD

**Issue by pomeroyk** [JuiceMind/JuiceMind#349]
Here's the unittest I'm trying to use in a python program with a getDigits( ) function:

def test_getDigits(self):
  hun,tens,one = getDigits(546)
  self.assertTrue(hun==5)
  self.assertEqual(tens,4)
  self.assertTrue(one == 6)

The test fails, even though the function code is correct.  Also, the results tab would be better if the results from the asserts was viewable.  Right now it's just frustrating, students aren't going to be able to use this. 

Thanks for any help!

**Comment by juicemind-dev** [comment]
Yes, we're adding the viewable results it's one of the next updates we are deploying.

Thanks for sending this example, it should be fixed shortly.
